src/services/request.service.js

```diff
@@ -96,11 +95,29 @@ const getManagerRequest = async (userId) => {
   return requests;
 };
 
+const updateRequestStatus = async (requestId, status) => {
+  if (status === 'approved' || status === 'declined') {
```

Comment:
`approved` and `declined` can be placed in an `enum` like structure.

Reply:
Thanks for this. Yeah, I have placed them in an `array` then on `if` I used `array.includes()` function to check if it is the status. What do you think

Reply:
Reading from an array is very expensive. You can use an object which will give us a time complexity of O(1)

Reply:
nice suggestion, I didn't think about that. An object can work